cross/make/Makefile

```
@@ -0,0 +1,16 @@
+PKG_NAME = make
+PKG_VERS = 4.4.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://ftp.gnu.org/gnu/make/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = https://ftp.gnu.org/gnu/make/
+COMMENT  = GNU Make is a tool which controls the generation of executables and other non-source files of a program from the program's source files
+LICENSE  = GPLv2
```

This has changed since 4.3...

```suggestion
LICENSE  = GPLv3+
```

---

sorry but I killed your commit by mistake...